zip-0316.rst

```diff
@@ -486,10 +486,10 @@ Requirements for both Unified Addresses and Unified Viewing Keys
   Receiver Encoding, as specified in the Zcash Protocol Specification
   [#protocol-nu5]_.
 
-* Producers SHOULD order the constituent Addresses/Viewing Keys in
-  the same order as in the Priority List above. However, Consumers
-  MUST NOT assume this ordering, and it does not affect which Address
-  should be used by a Sender.
+* Consumers MUST reject Unified Addresses/Viewing Keys in which the
```

Comment:
We don't necessarily need to add the rationale here, but it would be a good idea to put the rationale for this change into the commit message.